frontend/lib/configuration/settings_model.dart

```diff
@@ -16,12 +17,17 @@ class SettingsModel extends ChangeNotifier {
       _firstStart = await loadFirstStart();
       _hideVerificationInfo = await loadHideVerificationInfo();
       _locationFeatureEnabled = await loadLocationFeatureEnabled();
+      _staging = await loadStaging();
     } on Exception {
       _preferences?.clear();
     }
     return this;
   }
 
+  Future<void> clearPreferences() async {
+    _preferences?.clear();
```

Comment:
I think we should 
* Reset all local fields
* notify the listeners
* await the `clear` promise

Reply:
added `notify` and await not sure what else to reset is missing?

Reply:
I think we need to reset
      _firstStart 
      _hideVerificationInfo 
      _locationFeatureEnabled 
     _staging

Reply:
But isn't this done by the clear function?

Reply:
Okay i'm probably lacking some basic concepts of flutter...
Every time the `notifyListeners()` was called the settings gonna be `initialize()` because the `FutureBuilder<Settings>` recognizes that.
So when i `setEnableStaging` the `SettingsModel` gonna get `initialize()` so even if i reset these fields you describe they gonna be overwritten immediately in the `initialize` fkt and set to whatever in the loadFunctions is marked as default value . So i don't get the point of doing that tbh

Reply:
> Okay since the notifyListeneres triggers another initialize of the SettingsModel

I'm surprised about this, and I believe/hope this should not be true.

> why the default values and the fallback values differ.

That's weird indeed. Imo, the default values are not relevant (as we initialize them) . We might be able to get rid of them by using the `late` keyword.